src/components/templates/AnimateMap/AnimateMap.tsx

```diff
@@ -28,6 +29,10 @@ export const AnimateMap: React.FC<AnimateMapProps> = ({ venue }) => {
   const store = useStore();
   const user = useUser();
 
+  // TODO: Use this data to display venues
+  const rooms = useRelatedPartymapRooms({ venue });
+  console.log(rooms);
```

Comment:
let's remove it

Reply:
done by @yarikoptic

Reply:
Oh, was I useful this Friday? Cool

Reply:
@yarikoptic ahaha, sorry. Tagged the wrong person. Have great weekends!